spec/models/favorite_pitches_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe FavoritePitch, type: :model do
+  it "is valid with valid attributes" do
+    user = create(:user)
+    football_pitch = create(:football_pitch)
+    favorite_pitch = create(:favorite_pitch)
+    expect(favorite_pitch).to be_valid
+  end
+
+  it "requires a user" do
```

Comment:
sao e k dùng các validate hỗ trợ sẵn như
` RSpec.describe Booking, type: :model do
     context "associations" do
         it {is_expected.to belong_to :user}
     end
     context "validates" do
         it {is_expected.to validate_presence_of :booking_from}
     end
 end`